core-web/libs/dotcms-scss/angular/dotcms-theme/components/_button.scss

```diff
@@ -162,25 +162,54 @@
     padding-right: $spacing-2;
 }
 
-.p-button.p-button-icon-only.p-button-sm {
-    height: $spacing-4;
+.p-button.p-button-icon-only.p-button-text.p-button-sm {
+    height: $spacing-5;
+    width: $spacing-5;
+    gap: 0;
+
     border: 0 none;
-    color: $color-palette-primary;
-    opacity: 0.5;
+    background-color: transparent;
 
     &:enabled {
         &:hover {
-            opacity: 1;
+            background-color: $color-palette-primary-100;
         }
 
-        &:active,
+        &:active {
+            background-color: $color-palette-primary-200;
+        }
         &:focus {
-            opacity: 1;
+            box-shadow: 0px 0px 0px 2.8px #97aef6;
```

Comment:
We should use the `box-shadow` variables we already have. However, if needed, we can create a new variable.

Reply:
@fmontes what do you think about this?